datanode/gateway/graphql/schema.graphql

```diff
@@ -27,10 +27,10 @@ type PeggedOrder {
 
 "Details of the iceberg order"
 type IcebergOrder {
-  "Size of the order that will be made visible if the iceberg order is refreshed after a trade has occurred"
-  initialPeakSize: String!
-  "Threshold at which the order's visible remaining size will be refreshed back to its initial peak size"
-  minimumPeakSize: String!
+  "Size of the order that will be made visible if the iceberg order is replensished after trading"
+  peakSize: String!
+  "If the visible size of the order falls below this value it will be replensished back to the peak size using the reserved amount back to the peak size"
```

Comment:
```suggestion
  "Size of the order that will be made visible if the iceberg order is replenished after trading"
  peakSize: String!
  "If the visible size of the order falls below this value, it will be replenished back to the peak size using the reserved amount"
```